### PR TITLE
@craigspaeth: refactor target campaign url to pull from env variables instead of a hardcoded string

### PR DESCRIPTION
**Pull request by xtina-starr**
This is a follow up refactor to the [suggestion](https://github.com/artsy/force/pull/1670#discussion_r129929474) you made in a previous PR that suppressed the Marketing Signup Modals due to a facebook signup issues.

### CODE REVIEW

**Comment by craigspaeth** [comment]
🙌  thanks! LGTM. Feel free to merge when 💚

File: desktop/config.coffee [hunk 1]
```diff
@@ -97,6 +97,7 @@ module.exports =
   SITEMAP_BASE_URL: 'http://artsy-sitemaps.s3-website-us-east-1.amazonaws.com'
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$'
+  TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
```
Comment (xtina-starr) [inline desktop/config.coffee]: I did have one question. Should this value be set to null in the config file? I'm going to set them as environment variables regardless, but I was wondering would it be more appropriate to it to null here?

File: mobile/components/marketing_signup_modal/middleware.coffee [hunk 2]
```diff
@@ -4,10 +4,8 @@ module.exports = (req, res, next) ->
   sd = res.locals.sd
   slug = req.query['m-id']
 
-  # This is just for the Seattle Art Fair campaign and should be removed after
-  # the fair closes.
-  # (commenting this out for now because the facebook sign up is throwing an error)
-  # slug = 'ca3' if !slug and sd.CURRENT_PATH is '/seattle-art-fair-2017'
+  # Launches marketing signup modal for targeted url
+  slug = 'ca3' if !slug and sd.CURRENT_PATH is sd.TARGET_CAMPAIGN_URL
```
Comment (xtina-starr) [inline mobile/components/marketing_signup_modal/middleware.coffee]: Same here..

File: mobile/config.coffee [hunk 3]
```diff
@@ -57,6 +57,7 @@ module.exports =
   SESSION_COOKIE_MAX_AGE: 31536000000
   SESSION_SECRET: 'artsyoss'
   STRIPE_PUBLISHABLE_KEY: null
+  TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
```
Comment (xtina-starr) [inline mobile/config.coffee]: same here...

File: desktop/config.coffee (hunk 1 above)
Comment (craigspaeth) [inline desktop/config.coffee]: Nah, this is good. The values here are just used as development defaults. So it's nice to have something in there to serve as an example when appropriate.

Comment (xtina-starr) [inline desktop/config.coffee]: got it, thanks!

**Pull request merged**